assets/js/search_results.js

```diff
@@ -78,6 +81,19 @@ function planToView(plan, planIndex) {
     return view;
 }
 
+// The autocompleted locations need to be fixed. City and country names need to be capitalized and admin level 2 names need to be changed to all upper cases.
+function correctLocationName(location) {
```

Comment:
nit: call it normalizeLocation, since we just align the format to make things consistent

Reply:
Done